js/classes/entities/ui_ground/start_menu.js

```diff
@@ -75,9 +75,7 @@ class StartMenu {
         GAME.mode = GameEngine.GAMEPLAY_MODE;
         HUD.addComponents();
         this.removeFromWorld = true;
-        // setTimeout(() => {
-        //     ASSET_MGR.playMusic(MUSIC.PEACEFUL_CHIPTUNE.path, MUSIC.PEACEFUL_CHIPTUNE.volume);
-        // }, 1000);
```

Comment:
I do not understand how this fixes the issue. Is this where the problem with playing music at the start of the game regardless of audio "autoplay" being enabled was solved? Can you explain?